File: java/logica_y_validaciones/validar_si_contacto_existe.java
package logica_y_validaciones;
public class validar_si_contacto_existe {
    public static void main(String[] args) {
        String contactoGuardado = "Juan Pérez";
        String contactoBuscado = "Juan Pérez";
        if (contactoGuardado.equals(contactoBuscado)) {
            System.out.println("El contacto existe.");
        } else {
            System.out.println("El contacto no existe.");
        }
    }
}

File: visual_basic/condicionales/validar_si_contacto_existe/validar_si_contacto_existe.vb
Imports System
Module validar_si_contacto_existe
    Sub Main(args As String())
        Dim nombre As String = "Juan Pérez"
        If nombre <> "" Then
            Console.WriteLine("El contacto existe: " & nombre)
        Else
            Console.WriteLine("El contacto no existe")
        End If
        Console.ReadLine()
    End Sub
End Module